app/dashboard/page.tsx

```diff
@@ -40,17 +30,28 @@ export default async function Dashboard() {
     redirect('/login');
   }
   
+  // TODO(SWE-67): Grab posts from following users
+  // Sends a request to load the initial posts
+  const payload = { "userIds": ['dtran', 'jfales', 'sfales'] };
+  const res = await fetch('http://localhost:3000/api/posts/users?page=0&pageSize=5', {
+    method: 'POST',
+    headers: {'Content-Type': 'application/json'},
+    body: JSON.stringify(payload),
+  });
+
+  const posts = (await res.json())['posts'];
+
   return (<>
     <Navbar username={username}></Navbar>
     <main className='bg-cream-100 flex items-center justify-center pt-14'>
-      <div className='flex-[0_1_670px] flex flex-col items-center'>
+      <div className='flex-[0_1_670px] flex flex-col items-center pb-7'>
         <CreatePostModal/>
         {
-          fetchPosts(0, testData).map((post, index) => (
-            <Post name={post['name']} key={index}></Post>
+          posts.map((post, index) => (
+            <Post imageUrl={post[0]} title={post[1]} createdBy={post[6]} key={index}></Post>
```

Comment:
Replace array references with constant values for readability.

Before:
`imageUrl={post[0]}`

After: 
`const IMAGE_URL_INDEX = 0;`
`imageUrl={post[IMAGE_URL_INDEX]}`

This helps with readability and easier to update in the event a post's structure changes.